layouts/user.vue

```diff
@@ -0,0 +1,85 @@
+<script setup>
+const isOpen = ref(true)
+const links = [{
+  label: 'Moje szklarnie',
+  icon: 'i-heroicons-map-pin-solid',
+  to: '/'
+}, {
+  label: 'Moje lokalizacje',
+  // placeholder bo nie wiem skad wziac ikonke szklarni
+  icon: 'i-heroicons-building-storefront-solid',
+  to: '/'
+}, {
+  label: 'Mój profil',
+  icon: 'i-heroicons-user-circle-solid',
+  to: '/'
+}]
+</script>
+<template>
+  <div>
+    <UButton
+      class="my-1"
+      color="gray"
+      size="xl"
+      icon="i-heroicons-arrow-right-20-solid"
+      variant="ghost"
+      @click="isOpen = true"
+    />
+    <div class="relative">
+      <div class="absolute -top-9 right-0">
+        <UAvatar
+          src="https://avatars.githubusercontent.com/u/739984?v=4"
+          alt="Avatar"
+        />
+      </div>
+    </div>
+    <USlideover
```

Comment:
`<USlideover>` będzie zawsze nad treścią. Nie wiem czy to dobry pomysł by go tutaj zastosować